src/services/store.ts

```diff
@@ -23,14 +23,27 @@ export type Actions = {
   providers: Provider[]
 
   /**
-   * Representative points, fetched and cached given a `distribution` and `serviceArea`.
+   * Representative points ("RP"), fetched and cached given a `distribution`
+   * and `serviceArea`.
+   *
+   * For each service area, we sample the real population (from USPS & census data).
+   * The larger the `distribution` parameter, the larger each RP becomes, and the
+   * fewer RPs we generate per service area.
    */
   representativePoints: RepresentativePoint[]
 
   /**
+   * Service areas that the user selected on the map.
+   *
+   * We support just one selection for now, but will likely support
+   * multi-select in the future.
+   */
+  selectedServiceArea: string | null
+
+  /**
    * Strings representing county-zip tuples selected by the user in the
    * Service Area Drawer. For example, `["Yolo", "95606"]` maps to the
-   * string `"Yolo-95606"`. This representation makes it cheap to compare
+   * string `"Yolo / 95606"`. This representation makes it cheap to compare
```

Comment:
Being picky, but can you use a real name for clarity?

Reply:
What would the people of Yolo County say to you? https://github.com/bayesimpact/tds-frontend/blob/9bc18e78186cdf58f857fa167f882b6e0703484d/src/constants/zipCodes.ts#L62

Reply:
😂